protos/manual_control/manual_control.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
+
+package mavsdk.rpc.manual_control;
+
+option java_package = "io.mavsdk.manual_control";
+option java_outer_classname = "Manual_ControlProto";
+
+service ManualControlService
+{
+
+    rpc ConnectDrone (ConnectRequest)  returns (ConnectResponse) {}
+    /*
+     * Send command to enable Joystick mode 
+     *
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    
+    rpc SetPositionControl (PositionControlRequest)  returns(PositionControlResponse) {}
+    
+    
+    rpc SetAltitudeControl (AltitudeControlRequest)  returns(AltitudeControlResponse) {}
+    
+
+
+    rpc ArmDrone (ArmRequest) returns (ArmResponse) {}
+
+
+    rpc DisarmDrone (DisarmRequest) returns (DisarmResponse) {}
+
+      /*
+     * Send command to command to the drone to move forwards or backwards 
+     */
+    rpc Axismotion (MotionRequest) returns (MotionResponse) {}
+
+     
+
+
+
+}
+
+message ConnectRequest{
+    int32 target =1 ;
+}
+
+message ConnectResponse{
+    ManualControlResult mc_response = 1;
+}
+
+message PositionControlRequest{}
+message PositionControlResponse{
+    ManualControlResult mc_response = 1;
+}
+
+message AltitudeControlRequest{}
+message AltitudeControlResponse{
+    ManualControlResult mc_response = 1;
+}
+
+message ArmRequest {
+    uint32 armedbutton =1; 
+}
+message ArmResponse {
+    ManualControlResult mc_response = 1;
+}
+
+message DisarmRequest {
+    uint32 disarmedbutton =1; 
+}
+message DisarmResponse {
+    ManualControlResult mc_response = 1;
+}
+
+
+message MotionRequest {
+    sint32 xmotion = 1 ;                    // value between -1000 to 1000. negative-> backwards, positive-> forwards 
+    sint32 ymotion = 2 ;                    // value between -1000 to 1000. negative-> left, positive--> right
+    sint32 zmotion = 3 ;                    // value between -1000 to 1000. negative-> down ,positive--> up 
+    sint32 revolve = 4 ;                    // value between -1000 to 1000. negative-> ccw ,positive- -> cw  revolve aroun z-axis 
+}
+message MotionResponse{
+    ManualControlResult  mc_response = 1  ;
```

Comment:
Because of how auto-generation works, the `ManualControlResult` has to be called `manual_control_result`:

```suggestion
    ManualControlResult  manual_control_result = 1  ;
```

Reply:
done